GPU/Directx9/TextureCacheDX9.cpp

```diff
@@ -1633,7 +1577,7 @@ void *TextureCacheDX9::DecodeTextureLevel(GETextureFormat format, GEPaletteForma
 			}
 		} else {
 			tmpTexBuf32.resize(std::max(bufw, w) * h);
-			finalBuf = UnswizzleFromMem(texptr, bufw, h, 4);
+			finalBuf = UnswizzleFromMem(texptr, tmpTexBuf32, bufw, h, 4);
```

Comment:
Hmm, maybe we just move this buffer to the Common class?  We'll probably always have a need to depal or something...

-[Unknown]